build.sbt

```diff
@@ -4,35 +4,60 @@ name := "sbt-git-versioning"
 organizationName := "Rally Health"
 organization := "com.rallyhealth.sbt"
 
-enablePlugins(SemVerPlugin)
+// enable after SemVerPlugin supports sbt-plugins
+//enablePlugins(SemVerPlugin)
```

Comment:
This was failing because it couldn't find the 1.0.0 release. I'm guessing that I need to release 1.1.0 with these new settings, then I can try to turn this on.

Reply:
I have a fast-follow to upgrade this plugin to use sbt 1.0 itself. I'll add this back for that PR (or before, since I might also fast-follow with code coverage).